Portal/src/Datahub.Portal/Pages/Project/DataProjects.razor

```diff
@@ -73,7 +73,7 @@
         await using var context = await _dbFactory.CreateDbContextAsync();
         _allProjects = context.Projects
             .Where(p =>
-                (p.Project_Status_Desc == Datahub_Project.ONGOING || p.Is_Featured) &&
+                (p.Project_Status == 1 || p.Is_Featured) &&
```

Comment:
Need to move out of magic number territory

Reply:
created bug for Nabeel